Jenkinsfile

```diff
@@ -322,27 +322,21 @@ pipeline {
                 }
             }
         }
-        stage('Get Commit Message') {
-            steps {
-                script {
-                    env.COMMIT_MESSAGE = sh(script: 'git show -s --format=%B',
-                                            returnStdout: true).trim()
-                    Map pragmas = [:]
-                    // can't use eachLine() here: https://issues.jenkins.io/browse/JENKINS-46988/
-                    env.COMMIT_MESSAGE.split('\n').each { line ->
-                        String key, value
-                        try {
-                            (key, value) = line.split(':', 2)
-                            if (key.contains(' ')) {
-                                return
-                            }
-                            pragmas[key.toLowerCase()] = value
-                        /* groovylint-disable-next-line CatchArrayIndexOutOfBoundsException */
-                        } catch (ArrayIndexOutOfBoundsException ignored) {
-                            // ignore and move on to the next line
+        stage('Prepare Environment Variables') {
+            // TODO: Could/should these be moved to the environment block?
+            parallel {
+                stage('Get Commit Message') {
+                    steps {
+                        pragmasToEnv()
+                    }
+                }
+                stage('Determine Release Branch') {
+                    steps {
+                        script {
+                            env.RELEASE_BRANCH = releaseBranch()
```

Comment:
How does `releaseBranch` call `get_release_branch`?

Reply:
Heh.  https://github.com/daos-stack/daos/blob/bead33473c2aaa9b93645f0122021188df4f6a14/Jenkinsfile#L141-L150

Reply:
Sorry if double posted, but it seems GH lost some of my comments...

This
`daos:/Jenkinsfile`
calls
`pipeline-lib:/vars/releaseBranch.groovy`
which calls
`daos:/utils/rpms/packaging/get_release_branch`

IMO that's messy to bounce back and forth between repos